Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -769,7 +769,7 @@ static void dispatchSyntheticMouseMove(LocalFrame& mainFrame, const WebCore::Flo
     auto isFirstSyntheticClickOnPage = !m_hasHandledSyntheticClick;
```

Comment:
I need to remove this instance too, as its only usage is in the context of this quirk.

Reply:
done.